packages/quantic/force-app/main/default/lwc/quanticSearchBox/templates/defaultSearchBox.html

```diff
@@ -0,0 +1,68 @@
+<!-- sldsValidatorIgnore -->
```

Comment:
Why do we need this?

Reply:
It was asking if I wanted to ignore the "mobile" slds validation... Guess it just adds this to ignore ALL validations

Reply:
It just highlights pretty much all the HTML with warnings so it's pretty useless.